siriuspy/siriuspy/sofb/csdev.py

```diff
@@ -791,18 +790,6 @@ def get_sofb_database(self, prefix=''):
             'LoopPerfTimGetOStd-Mon': {
                 'type': 'float', 'value': 0, 'unit': 'ms', 'prec': 1,
                 'lolim': -1, 'hilim': 100},
-            'LoopPerfTimGetKMax-Mon': {
```

Comment:
@fernandohds564 , @anacso17 , we should remember to take all these removed PVs from archiver.